test/meson.build

```diff
@@ -40,7 +40,7 @@ if enable_ccapi
   subdir('unittest')
 endif
 
-if get_option('enable-nnstreamer-tensor-filter')
+if get_option('enable-nnstreamer-tensor-filter') and not tizen_6_0
```

Comment:
Is this because of the dependency on ```nnstreamer-test-devel``` package? If so, it appears that you need to check it with meson's dependency check (or by checking the file existance).

Reply:
this will be removed after tensorflow2 spec in tizen 6.0 is fixed.